duckduckgo/bloom_cpp
# make swift package

Task/Issue URL: https://app.asana.com/0/903182762228930/1202443550575711/f
Tech Design URL: 
CC: 

**Description**:

Make this a swift package so we go back to having a single source of truth. 

**Steps to test this PR**:
1. Test https://github.com/duckduckgo/BrowserServicesKit/pull/184 
2. Ensure local tests run (run_test.sh)

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
